deploy/20_cluster_config_v1_reader_role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: cluster-config-v1-reader
```

Comment:
As the managed-velero-operator creates this exact named role, will this conflict?

Reply:
I was thinking about that same thing, and I'm not sure. Both operators need to ensure that the role is there, but can both try to create it in the same way?

Reply:
I actually think I need to also update the selectorsyncset here.